packages/@lwc/engine-core/src/shared/create-caching-css-generator.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+
+import { StylesheetFactory, StylesheetFactoryResult } from './stylesheet-factory';
+import { hasAdoptedStyleSheets } from './has-adopted-stylesheets';
+import { isUndefined } from '@lwc/shared';
```

Comment:
Move the bare imports above the relative ones.

Reply:
There's an ESLint rule that can do that ;) . But OK yes, will do.